dev/src/main/java/org/eclipse/codewind/intellij/ui/tree/CodewindToolWindowHelper.java

```diff
@@ -24,12 +28,61 @@
     public final static String SHOW_LOG_FILES_TOOLWINDOW_ID = message("LogFilesToolWindow");  // Note this ID is actually the UI displayed string in the ToolWindow
 
     public static void openWindow(Project project) {
+        CoreUtil.invokeLater(new Runnable() {
+            @Override
+            public void run() {
+                ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow(CodewindToolWindow.ID);
+                if (toolWindow != null && toolWindow.isAvailable()) {
+                    toolWindow.show(this);
+                    ContentManager contentManager = toolWindow.getContentManager();
+                    final Content content = contentManager.findContent(CodewindToolWindow.DISPLAY_NAME);
+                    JComponent component = content.getComponent();
+                    if (content != null && component instanceof CodewindToolWindow) {
+                        CodewindToolWindow codewindToolWindow = (CodewindToolWindow)component;
+                        codewindToolWindow.expandToProject(project);
+                    }
+                }
+            }
+        });
+    }
+
+    public static void openWindowAndWaitForContentToPopulate(Project project) {
         CoreUtil.invokeLater(new Runnable() {
             @Override
             public void run() {
                 ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow(CodewindToolWindow.ID);
                 if (toolWindow != null && toolWindow.isAvailable()) {
                     toolWindow.show(null);
+                    ContentManager contentManager = toolWindow.getContentManager();
+                    final Content content = contentManager.findContent(CodewindToolWindow.DISPLAY_NAME);
+                    System.out.println("content.isValid() = " + content.isValid());
```

Comment:
Do we need these debug statements to remain in the code?

Reply:
Definitely not.